docs/user_manual/getting_verified.md

```diff
@@ -32,21 +32,33 @@ because that server is where most game servers are listed.
 Getting registered for the login server run by Griffon26 is an automated process:
 
 1. First log in to the Hirez servers
-1. Join a game server
-1. Open the Chat Console. (This might not have a keybinding, add one through `Settings > Keybindings > Interface`). Note: This console is different to the Unreal Console (~)
-1. Send a private chat message to `@taserverbot` with only the word `authcode` in it:
 
-    `@taserverbot authcode`
+2. Join a game server
 
-1. If you get an error saying the player taserverbot is not online, message Griffon26 in Discord
-1. Now log in to Griffon26's login server ("the community servers") with the in-game
+3. Open the Chat Console by pressing `T` (Default keybinding. If it's missing, add one through
+   `Settings > Keybindings > Interface`). Note: This chat console is different to the
+   Unreal Console (~)
```

Comment:
They will have to press escape followed by `t`. `t` alone won't do the trick.
Write `t` in lower case so people don't think they have to use shift.
Remove the remark about keybinding or modify it to something like "If you have bound Chat to a different key than `t`, use that key instead".